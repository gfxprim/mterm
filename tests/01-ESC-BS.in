5 2
A\e[2\bCB
