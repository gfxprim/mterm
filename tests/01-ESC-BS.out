 -----
|A B  |
|     |
 -----
size 2x5 cursor 0x3
